modules/ServerAPI/src/client/SaltBrowser.html

```diff
@@ -109,84 +108,163 @@ <h5>Searching for Salts containing text "<i class="bv_exptSearchTerm"></i>"</h5>
     <!-- CREATE SALT SECTION -->
     <div class="modal hide bv_createSalt">
         <div class="modal-header">
-            <h3>Create Salt</h3>
+            <h3>Register New Salt</h3>
         </div>
         <div class="modal-body">
+            <!-- Input Salt Section-->
             <div class="bv_chemicalStructureForm">
             </div>
+            <p><i>Use the sketcher to sepcify the molecular structure of the salt. Pay close attention to changes as they will be used in calculating Lot Molecular Weight. Use the + and - buttons to add positive or negative charges</i></p>
```

Comment:
typo: `Pay close attention to changes` should be `Pat close attention to charges`